apps/webpack.js

```diff
@@ -149,7 +150,8 @@ var baseConfig = {
       '@cdo/apps': path.resolve(__dirname, 'src'),
       '@cdo/static': path.resolve(__dirname, 'static'),
       repl: path.resolve(__dirname, 'src/noop'),
-      '@cdo/storybook': path.resolve(__dirname, '.storybook')
+      '@cdo/storybook': path.resolve(__dirname, '.storybook'),
+      serialport: false
```

Comment:
This same `serialport: false` config is in `karmaConfig`, could we remove it there now?

There's also a few lines in karmaConfig that appear tied to serialport, do we have issues running tests now without them?

https://github.com/code-dot-org/code-dot-org/blob/ffe57dbc826c9e5b6238d706de93fe3e7cac9a77/apps/webpack.js#L394-L397

Reply:
Sure! Removed the lines and pushed to remote - build passed!